DiamondLightSource/SynchWeb
# [LIMS-1552] Remove old jQuery version

**JIRA ticket**: [LIMS-1552](https://jira.diamond.ac.uk/browse/LIMS-1552)

**Summary**:

A minified version of jQuery which was only used in one function was found, and since that function has little to no effect on the surrounding environment (logfiles), it may be possible to remove it completely.

I've tested Fast DP reports and autoPROC log files, and both of them don't see any penalty in terms of usability. AutoPROC files link resources as **b64 blobs**, so there are no authorised requests being made.

**Changes**:
- Remove JS code from log `iframe`

**To test**:
- Go to /dc/visit/cm37235-5, view logs for any autoprocessing program (autoPROC, fast_dp...), ensure that links are not broken with the removal of the JS script in the iframe

Note that the list of autoprocessing programs is not exhaustive, and other logfile types may be affected by this. I need help mapping out which logfiles could be affected.
